src/components/selectionBoxLayer.ts

```diff
@@ -111,6 +131,59 @@ export module Components {
     public fixedHeight() {
       return true;
     }
+
+    /**
+     * Gets the x scale for this SelectionBoxLayer.
+     */
+    public xScale(): QuantitativeScale<any>;
+    /**
+     * Sets the x scale for this SelectionBoxLayer.
+     * 
+     * @returns {SelectionBoxLayer} The calling SelectionBoxLayer.
+     */
+    public xScale(xScale: QuantitativeScale<any>): SelectionBoxLayer;
```

Comment:
The `any` here seems pretty iffy. Is there a way to parameterize on the type of scale?